pkg/cmd/operator/operator.go

```diff
@@ -295,7 +295,8 @@ func getOperatorImage(ctx context.Context, c ctrl.Reader) (string, error) {
 	ns := platform.GetOperatorNamespace()
 	name := platform.GetOperatorPodName()
 	if ns == "" || name == "" {
-		return "", nil
+		// We are most likely running out of cluster. Let's take a chance and use the default value
```

Comment:
Ditto

Reply:
Done see #5109. 
I have taken the liberty of bundling these two changes as:
* They are scope to the same issue 
* They are located in the same context/file
* They are small enough